### ISSUE TITLE
Server crash for new 1.10.2 1.0.0 update

### ISSUE THREAD

**Issue by Gamerbolts** [rwtema/extrautilities_old_issues#934]
Hi I am getting a crash on my server only with the new update. It looks like it has something to do with a client gui. Here is the crash log http://pastebin.com/3FQYLu09

Thanks for the help!

**Comment by whatthedrunk** [comment]
Getting the same thing, did you figure it out?

**Comment by Gamerbolts** [comment]
Nope, not yet. I've come across this multiple times with different mods and told the mod devs and basically it is a server mod calling for a client gui which it can't do so it crashes. So just have to wait for an update.

**Comment by JohnC94** [comment]
I came here looking for better news on this issue. Guess we're stuck waiting for an update. I'm having this problem as well.

**Comment by ghost** [comment]
same issue i am having with a gui issue causing the crash 

Version of EU 2 extrautils2-1.10.2-alpha-1.0.0

http://pastebin.com/FVZEpWxa

**Comment by Hastling** [comment]
Same issue as everyone else. 

http://pastebin.com/k7WCSJuS

**Comment by Gamerbolts** [comment]
Thank you RWTema! The new 1.0.1 update has fixed the server crash
